frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -350,6 +422,79 @@ class ConcatenateWith {
       bridge.handleSomeStaticStreamSinkSingleArgStaticMethodConcatenateWith(hint: hint);
 }
 
+@freezed
+class CustomError with _$CustomError implements FrbException {
+  @Implements<FrbBacktracedException>()
```

Comment:
Also, we can assert in our tests that the thrown error `isA<FrbBacktracedException>`